test/unit/UnitTestRouter.ts

```diff
@@ -133,9 +143,16 @@ describe('Router Unit Tests', () => {
         assert.equal(await router.claimWindow(tx), TxStatus.FreezePeriod);
       });
 
-      it('returns ClaimWindow when claim window started', async () => {
+      it('returns ClaimWindow when claim window started and claiming disabled', async () => {
         const tx = mockTxStatus(txTimestamp, TxStatus.ClaimWindow);
+        config.claiming = false;
         assert.equal(await router.claimWindow(tx), TxStatus.ClaimWindow);
+        config.claiming = true;
+      });
+
+      it('returns ClaimWindow when claim window started and claiming disabled', async () => {
```

Comment:
Should be `'returns ClaimWindow when claim window started and claiming ENABLED'`?